test/Table_Tests/src/In_Memory/Table_Running_Spec.enso

```diff
@@ -277,6 +313,11 @@ add_specs suite_builder =
             # 4 | SG0456 | E         | 73.77        | 73.77
             expected_table = table.zip expected_column
             result.should_equal expected_table
+            warnings = Problems.get_attached_warnings result
+            warnings.not_empty . should_be_true
+            warnings.first.should_be_a Ignored_Nothing_Values
```

Comment:
We should definitely not rely on ordering of warnings (if any other warning here is possible; if not `not_empty` should probably be replaced with `warnings.length . should_equal 1`).

But overall, I'd prefer to rely on `Problems.expect_warning` if a warning is expected and other warnings are acceptable; and `Problems.expect_only_warning` if only a specific warning is expected and no others.

Both methods _return_ the expected warning if it is found, so we can do `w = Problems.expect_warning Type result` and then do additional checks on `w` like `w.column . should_equal "Ticket Price"`

Reply:
Done